quarkc/backend.py

```diff
@@ -217,8 +219,9 @@ def genmain(self):
         path = self.add_import(self.main)
         self.files[fname] += self.gen.main(path, self.name(self.main.name))
 
-    def genimps(self, imps):
-        imps = [self.gen.import_(pkg, org, dep) for (pkg, org, dep) in imps]
+    def genimps(self, imps, lazy=False):
```

Comment:
I'd like to improve the code's readability every time we touch something, so could you add a docstring every time you update a method's signature?